apps/src/lib/cli.rs

```diff
@@ -142,6 +142,7 @@ pub mod cmds {
                 .subcommand(TxUpdateVp::def().display_order(1))
                 .subcommand(TxInitAccount::def().display_order(1))
                 .subcommand(TxInitValidator::def().display_order(1))
+                .subcommand(TxResult::def().display_order(1))
```

Comment:
I think this might be a bit better fix for the queries group 3 below

Reply:
Thanks! Good point. I'll rename `TxResult` to `QueryResult`, and place it in `display_order` 3.